projects/wvr-elements/src/lib/wvr-header/wvr-header.component.ts

```diff
@@ -17,6 +17,9 @@ export class WvrHeaderComponent {
   /** The header title value to be displayed as a page title. */
   @Input() headerTitle = 'Weaver Header Component';
 
+  /** A URL link clickable from the page title to landing page. */
+  @Input() headerTitleUrl = '';
```

Comment:
This needs to be undefined, not `''`

`@Input() headerTitleUrl;` instead of `@Input() headerTitleUrl = '';`

Reply:
Modified this line with : `@Input() headerTitleUrl: string`;